models/glossary.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import "sort"
+
+// The Glossary object is used to manage a Glossary in a db.Store
+type Glossary map[string]string
+
+// SortKeys will return a slice of ordered keys.
+func (g Glossary) SortKeyAlphabetical() []string {
```

Comment:
The Karma model already has a function exactly like this, so let's just follow that pattern:
```go
func (g Glossary) SortKeys(ascending bool) []string {
    keys := make([]string, 0, len(g))
    for key := range g {
        keys = append(keys, key)
    }

    if ascending {
        sort.Sort(sort.StringSlice(keys)
    } else {
        sort.Sort(sort.Reverse(sort.StringSlice(keys))
    }

    return keys
}
```

You'll also want to copy the [same test pattern](https://github.com/quintilesims/slackbot/blob/alias/models/karma_test.go).